package.json

```diff
@@ -19,7 +19,9 @@
     "@fortawesome/free-solid-svg-icons": "^6.2.1",
     "@fortawesome/react-fontawesome": "^0.2.0",
     "@mui/icons-material": "^5.11.0",
+    "@mui/lab": "^5.0.0-alpha.114",
```

Comment:
We should not be using mui/lab as these are beta components 
Please use some alternative from mui package.